src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/AnyOS/AsnHelpers.cs

```diff
@@ -127,14 +128,20 @@ internal static AlgorithmIdentifier ToPresentationObject(this AlgorithmIdentifie
                 case Oids.TripleDesCbc:
                     keyLength = KeyLengths.TripleDes_192Bit;
                     break;
+                case Oids.RsaOaep when !asn.HasNullEquivalentParameters():
```

Comment:
This is so that the `KeyEncryptionAlgorithm` on `KeyTransRecipientInfo` contains the parameters. This isn't strictly necessary but it it seems correct to do so now that the parameters can be meaningful, and makes it easier to test that the intended RSA mode was actually used.